src/style.css

```
@@ -0,0 +1,144 @@
+* {
+  margin: 0;
+  padding: 0;
```

```suggestion
  box-sizing: border-box;
```

don't reset margin and padding for all. It's very difficult  process.

```
h1, h2, p, {
  margin: 0;
  p: 0;
}
```